python_modules/dagster/dagster/_core/host_representation/external_data.py

```diff
@@ -1545,7 +1545,13 @@ def _get_nested_resources(
             )
             for k, nested_resource in resource_def.nested_resources.items()
         }
-        if isinstance(resource_def, (ConfigurableResource, PartialResource))
+        if isinstance(
```

Comment:
handles anon. nested resources (only in play with configurableresources)

Reply:
can you make a comment in the code